packages/flame/lib/src/collisions/collision_detection.dart

```diff
@@ -70,9 +70,17 @@ abstract class CollisionDetection<T extends Hitbox<T>> {
   /// Returns the first hitbox that the given [ray] hits and the associated
   /// intersection information; or null if the ray doesn't hit any hitbox.
   ///
+  /// [ignoreHitboxes] can be used if you want to ignore certain hitboxes, i.e.
+  /// the rays will go straight through them. For example the hitbox of the
+  /// component that you might be casting the rays from.
+  ///
   /// If [out] is provided that object will be modified and returned with the
   /// result.
-  RaycastResult<T>? raycast(Ray2 ray, {RaycastResult<T>? out});
+  RaycastResult<T>? raycast(
+    Ray2 ray, {
+    List<ShapeHitbox>? ignoreHitboxes,
```

Comment:
Should it be `List<T>` here (and in 2 methods below)?

Reply:
True, copy paste error!